### PR TITLE
Col #38 Edit contact details

### PR DESCRIPTION
**Pull request by remaininlight**
#### Issues this PR relates to
https://github.com/yalla-coop/cost-of-living-support/issues/38
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Comment by remaininlight** [comment]
Added google-libphonenumber because the client wouldn't compile without it 🤷‍♂️

File: client/package.json [hunk 1]
```diff
@@ -103,6 +102,7 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.10",
+    "google-libphonenumber": "^3.2.31",
```
Comment (fadeomar) [inline client/package.json:105]: this should be in the dependencies not on devDependencies because we need it in the production

**Review by fadeomar — CHANGES_REQUESTED** [review]
hey @remaininlight nice work but I've some notes: 
  - [ ] the page title should be change to (Edit account details)
  - [ ] the tip card that says (Please note that changing your unique link will mean we need to review your profile again ) need to be added before the button https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=4617%3A19083&t=da3LfZoEdwgaf199-0 
  - [ ] the space between the page title and the (Below you can edit details for your customers such as your adviser contact line) line should be 32px 
image
  - [x] please tick the points that you have done in your PR in the original issue https://github.com/yalla-coop/cost-of-living-support/issues/38

**Comment by remaininlight** [comment]
> * the tip card that says (Please note that changing your unique link will mean we need to review your profile again )

Thanks for the feedback @fadeomar. This tip card has been outwitting me for a while, I've tried *exactly* the same code for this file as on CoL but there are still differences in the way the tip card displays:

Cost of Living:
image

Universal Credit:
image

Do you know why this might be please?

**Pull request merged**